packages/iris-grid/src/sidebar/visibility-ordering-builder/sortable-tree/TreeItem.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/prop-types */
+import React, { forwardRef, HTMLAttributes } from 'react';
+import classNames from 'classnames';
+import styles from './TreeItem.module.scss';
+import { IrisGridTreeItem } from './utilities';
+
+export interface Props extends Omit<HTMLAttributes<HTMLLIElement>, 'id'> {
+  childCount?: number;
+  clone: boolean;
+  collapsed?: boolean;
+  depth: number;
+  disableInteraction?: boolean;
+  disableSelection?: boolean;
+  ghost?: boolean;
+  handleProps?: any;
+  indicator?: boolean;
+  indentationWidth: number;
+  value: string;
+  item: IrisGridTreeItem;
+  onCollapse?(): void;
+  onRemove?(): void;
+  wrapperRef?(node: HTMLLIElement): void;
+  renderItem(props: {
+    clone: boolean;
+    childCount: number;
+    value: string;
+    item: IrisGridTreeItem;
+  }): React.ReactNode;
+}
+
+export const TreeItem = forwardRef<HTMLDivElement, Props>((props, ref) => {
+  const {
+    childCount = 0,
```

Comment:
Unused prop... are you trying to assign default values to the props here? I think would need to do in the `renderItem` function defined in `VisibilityOrderingBuild` instead.